src/main/kotlin/org/rust/ide/surround/RustSurroundDescriptor.kt

```diff
@@ -0,0 +1,27 @@
+package org.rust.ide.surround
+
+import com.intellij.lang.surroundWith.SurroundDescriptor
+import com.intellij.lang.surroundWith.Surrounder
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import com.intellij.psi.PsiWhiteSpace
+import com.intellij.psi.util.PsiTreeUtil
+import org.rust.lang.core.psi.RustExpr
+
+class RustSurroundDescriptor : SurroundDescriptor {
+
+    val surrounders = arrayOf(
```

Comment:
can you point out the reason why this array should be in a companion object? @jajakobyly